draft-ietf-tls-esni.md

```diff
@@ -324,10 +324,10 @@ cipher_suite
 provided in the corresponding `ECHConfig.cipher_suites` list.
 
 config_digest
-: A cryptographic hash of the `ECHConfig` structure from which the ECH key was
-obtained, i.e., from the first byte of "version" to the end of the structure.
-This hash is computed using the hash function associated with `cipher_suite`,
-i.e., the corresponding HPKE KDF algorithm hash.
+: Equal to `Extract(Expand(salt, config), "ech_config_digest", Nh)`, where
+`salt` is a string of `Nh` bytes set to `0`, `config` is the `ECHConfig`
+structure, and `Extract`, `Expand`, and `Nh` are as defined by the KDF API in
+{{!I-D.irtf-cfrg-hpke}}.
```

Comment:
Rather than repeat the definition here, what you probably want to do is make a new section under {{outer-extensions}} and cite that.  It's reasonable to do that as a follow-up though.

Reply:
Let's do it in follow up.

My view is that the hash should be dropped from outer_extensions: see #263.